src/autora/doc/pipelines/data.py

```diff
@@ -0,0 +1,21 @@
+from typing import Iterable, List, Tuple
+
+
+def load_data(data_file: str) -> Tuple[List[str], List[str]]:
+    import jsonlines
+
+    with jsonlines.open(data_file) as reader:
+        items = [item for item in reader]
+        inputs = [f"{item['instruction']}" for item in items]
```

Comment:
Why not just do `str(item['instruction'])` for a string casting? this is not as clear this way.

Reply:
I think this used to be a more complex string and it whittled down to this. You are right, I'll change it.